wikimedia/wikipedia-ios
# [YiR] V2 - Saved Articles slide

**Phabricator:** https://phabricator.wikimedia.org/T376067

### Notes
* Adds the saved count slide for eligible users (logged-in users with more than 3 saved articles)
* Shows collective slide for the rest of users
* Saved count and the random saved article titles are retrieved from the main app core data and passed through an async call, using a delegate
* These new core data methods live in MWKSavedPageList
* Copy and assets are still outdated, no V2 updates were made

### Test Steps
1. Logged in, device set to Italy or Mexico, set main language to en, es or it
2. Browse 7+ articles
3. Background the app 
4. You should see the personalized saved article slide if you have more than 3 saved articles, plus the names of 3 random articles you saved
5. Remove saved article, leave 2 or less
6. Trigger the regeneration of the report (you can disable and re-enable the YiR repot in settings)
7. You should now see the collective slide
8. Logged out, in a device with any number of saved slides - you should always see the collective slide
